waiter/src/waiter/handler.clj

```diff
@@ -330,15 +336,19 @@
           response-data (map
                           (fn service-id->service-info [service-id]
                             (let [scaling-state (retrieve-scaling-state query-autoscaler-state-fn service-id)
-                                  service-description (service-id->service-description-fn service-id :effective? false)
-                                  source-tokens-entries (service-id->source-tokens-entries-fn service-id)]
+                                  core-service-description (service-id->service-description-fn service-id :effective? false)
+                                  source-tokens-entries (service-id->source-tokens-entries-fn service-id)
+                                  instance-counts (retrieve-instance-counts service-id)
+                                  effective-service-description (service-id->service-description-fn service-id :effective? true)
+                                  usage-map (compute-service-usage effective-service-description (reduce + (vals instance-counts)))]
                               (cond->
-                                {:instance-counts (retrieve-instance-counts service-id)
+                                {:instance-counts instance-counts
                                  :last-request-time (get-in service-id->metrics [service-id "last-request-time"])
                                  :service-id service-id
-                                 :service-description service-description
+                                 :service-description core-service-description
                                  :status (service/retrieve-service-status-label service-id global-state)
-                                 :url (prepend-waiter-url (str "/apps/" service-id))}
+                                 :url (prepend-waiter-url (str "/apps/" service-id))
+                                 :usage usage-map}
```

Comment:
Nitpick: I think you're right.  `:resource-usage` feels better to me.

Reply:
I considered naming this `:resources-requested` (or something similar) and and reserve `:resource-usage` to mean actual usage.

Reply:
Ultimately decided `:resources-requested` feels awfully k8s-specific.

Reply:
Renamed to `resource-usage`